lava_test_plans/projects/lkft-android/devices/dragonboard-845c

```diff
@@ -0,0 +1,35 @@
+{% extends "devices/dragonboard-845c" %}
+
+{% set TAGS = TAGS|default(["lcg"]) %}
+
+{# to avoid overriding by the one definied in testcases #}
+{# like the one definied in testcases/boot.yaml is true by default #}
+{% set auto_login = false %}
```

Comment:
Why do you want "auto_login = false" when you do a boot test only?

Reply:
if you check what that auto_login is used for, you would see it's used for the case to login with some username, which is not for the Android case, Android does not need to input the username to login the serial console shell.